src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -268,9 +277,9 @@ private IEnumerable<ECPoint> GetCommitteeMembers(StoreView snapshot)
         [ContractMethod(1_00000000, CallFlags.AllowStates)]
         public ECPoint[] GetNextBlockValidators(StoreView snapshot)
         {
-            StorageItem storage = snapshot.Storages.TryGet(CreateStorageKey(Prefix_NextValidators));
+            StorageItem storage = snapshot.Storages.TryGet(CreateStorageKey(Prefix_NextCommittee));
             if (storage is null) return Blockchain.StandbyValidators;
-            return storage.GetSerializableList<ECPoint>().ToArray();
+            return storage.GetSerializableList<ECPoint>().Take(ProtocolSettings.Default.ValidatorsCount).OrderBy(p => p).ToArray();
```

Comment:
We should store the array sorted.

Reply:
The array is sorted by votes. Func like `GetValidators` also use this stored data.